---
layout: gnatdoc_error
crate: gnatcoll_python
version: 21.0.0
---
```

raised CONSTRAINT_ERROR : GNATdoc.Entities.Entity_Information_Maps.Constant_Reference: key not in map
ERROR: Command ["gnatdoc4", "-P", "python/gnatcoll_python.gpr", "--output-dir", "/home/chouteau/src/github/alire/docs.ada.dev/crate/gnatcoll_python/21.0.0", "--front-matter", "crate: gnatcoll_python", "--no-subprojects"] exited with code 1

```
